src/theme/colour-primitive/specs/pa-colour.ts

```diff
@@ -0,0 +1,95 @@
+import { PrimitiveColourSet } from "../../types";
+
+export const PAColourSet: PrimitiveColourSet = {
+    "brand-10": "#0E123A",
+    "brand-20": "#191E65",
+    "brand-30": "#232B90",
+    "brand-40": "#303BC5",
+    "brand-50": "#494CFF",
+    "brand-60": "#AD6EFF",
+    "brand-70": "#AEA4FF",
+    "brand-80": "#C5C4FF",
+    "brand-90": "#DDDCFF",
+    "brand-95": "#EDECFF",
+    "brand-100": "#F8F8FF",
+    "primary-10": "#FCF2F3",
+    "primary-20": "#f9e6e7",
+    "primary-30": "#f3cccf",
+    "primary-40": "#edb3b7",
+    "primary-50": "#e7999f",
+    "primary-60": "#e28087",
+    "primary-70": "#DC666F",
+    "primary-80": "#D0333F",
+    "primary-90": "#C4000F",
+    "primary-95": "#b0000e",
+    "primary-100": "#89000b",
```

Comment:
you might have reversed the colours by accident, should be darker for 10 and lighter for 100

![Screenshot 2025-03-03 at 6 01 36 PM](https://github.com/user-attachments/assets/b1839771-5f36-4172-a339-dc9b792ccde3)

Reply:
nooooooo

Reply:
fixed